examples/functions/fares/ENTUR-SchoolTwiceADayTripCarnet_2020120.xml

```diff
@@ -200,7 +200,7 @@
 						<AmountOfPriceUnitProduct id="ENT:school_carnet" version="1">
 							<Name lang="no">School transport with limited number of rides</Name>
 							<ChargingMomentType>beforeTravel</ChargingMomentType>
-							<OperatorRef versionRef="1.0" ref="ENT:bergen_muni"/>
+							<OperatorRef versionRef="EXTERNAl" ref="ENT:bergen_muni"/>
```

Comment:
Was there an issue with "1.0" ? related to the xmllint issue ?
And is the "l" instead of "L" in EXTERNAl a typo or for purpose ?

Reply:
The original problem was that xmllint suggested that something broke due to 'duplicate ref', but I am going to revert these things. Since it was not the reason why things broke.

`EXTERNAl` was the original typo, which I consistently applied.